inspire_matcher/utils.py

```diff
@@ -92,3 +91,29 @@ def get_tokenized_title(title):
 
     """
     return set(title.lower().split())
+
+
+def compute_title_score(x_title, y_title, threshold, math_threshold):
+    """Compute a score for a pair of titles.
+
+    Args:
+        x_title (string): one title.
+        y_title (string): the other title.
+        threshold (float): minimum overlap for the score to be non-zero in general.
+        math_threshold (float): minimum overlap for the score to be
+            non-zero in the presence of math in any of the titles.
+
+    Returns:
+        float: a score indicating the overlap of both titles, which is ``0`` if
+            below the threshold
+    """
+    x_title_tokens = get_tokenized_title(x_title)
+    y_title_tokens = get_tokenized_title(y_title)
+    current_title_jaccard = compute_jaccard_index(x_title_tokens, y_title_tokens)
+
+    some_title_has_math = (
+        "<math>" in x_title or "$" in x_title or "<math>" in y_title or "$" in y_title
```

Comment:
wouldn't regex be better for that?

Reply:
I don't know, escaping `$` would make it less readable and titles are short, so there's not much of a performance gain.